### PR TITLE
ES6 Modules

### PR DESCRIPTION
**Pull request by shafiuyushawu**
#  **Changes made ♻️**

- Created module folder 💯 
- Moved all functions to modules and linked them to `index.js` 💯 
- Changed all functions to arrow functions 💯 
- Used the `luxion.js` module to display the time string 💯

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -22,12 +22,12 @@ <h1 class="text-3xl font-bold">Awesome books</h1>
         </nav>
     </header>
     <main class=" flex flex-col -[100%] mx-5 mt-5">
```
Comment (Felix45) [inline dist/index.html:24]: - [ Required ] Good job on refactoring your Apps code to use `ES-6` syntax, however, it appears that after refactoring your code the App is no longer working. I expected that when I click on the individual links I would be able to see the following.
   
   - [x] Add book form
   - [x] Contact details page
   - [x] A list of books on the homepage
   
-  It's important that while refactoring the code you also make sure that the Application works as expected. Please see the video below and kindly rectify this.

## Video when using the App
https://user-images.githubusercontent.com/16392046/220451172-e2e2be55-7da5-4272-845b-d49bb268e7b9.mp4

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @shafiuyushawu , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :heavy_check_mark: 

- [x] No linter errors :white_check_mark: 
- [x] GitHub flow is used correctly :white_check_mark:
- [x] Repository has a professional readme file :white_check_mark:  
- [x] Descriptive pull request title and summary :white_check_mark:   

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html (hunk 1 above)
Comment (shafiuyushawu) [reply dist/index.html:24]: @Felix45 

Everything works pretty fine at my end and I re-clone the repository and checked
I suggest you kindly checked perhaps you missed something.

Comment (shafiuyushawu) [reply dist/index.html:24]: Hello @Felix45 

Kindly review below link confirm.

Thank you!

https://drive.google.com/file/d/1TSRH-rllmRaTGMvKTrofr61wr4R3sgLJ/view?usp=share_link

Comment (Felix45) [reply dist/index.html:24]: Hi @shafiuyushawu, 
Are you online, we can set a zoom meeting and have a look at this together. I have checked the project again and it's not working.

Comment (Felix45) [reply dist/index.html:24]: I set up a call with the student and we found the cause of the bug.

**Review by didierganthier — APPROVED** [review]
Hi @shafiuyushawu,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

**Pull request merged**